docs/core/tools/project-json-to-csproj.md

```diff
@@ -265,23 +300,6 @@ dependency version of a project reference.
 </ItemGroup>
 ```
 
-#### type: platform
-```json
-{
-  "dependencies": {
-    "Microsoft.NETCore.App": {
-      "version": "1.1.0",
-      "type": "platform"
```

Comment:
I think this information is still interesting. To say that there is no need for type: platform in csproj and that it does not exist there.